CenterForOpenScience/ember-osf-web
# [ENG-4158] Node navigation

-   Ticket: [ENG-4158]

## Purpose

Move new node navigation from guid-node.files template to guid-node template. 

## Summary of Changes

1. Move the template
2. Fix styling of existing guid-node templates to work with new navigation
3. Add metadata link to navigation

## Screenshot(s)

Metadata:
<img width="1742" alt="Metadata" src="https://user-images.githubusercontent.com/6599111/201685078-f56d2e36-6d6d-4e61-9503-acdfdba33299.png">

Files before:
<img width="1761" alt="Files-before" src="https://user-images.githubusercontent.com/6599111/201684950-d5100b6c-c0a9-477a-b9f0-fd9bb9937023.png">

Files after:
<img width="1698" alt="Files-after" src="https://user-images.githubusercontent.com/6599111/201685011-84c66844-30cd-43ee-82f4-c7cec519b9a6.png">

Registrations before:
<img width="1761" alt="Registrations-before" src="https://user-images.githubusercontent.com/6599111/201685178-1ccdc8a3-e0d6-4542-9990-e8a7f4dd06ed.png">

Registrations after:
<img width="1742" alt="Registrations-after" src="https://user-images.githubusercontent.com/6599111/201685221-6f5351d4-2774-40d3-a13c-ea8134fc41c2.png">

Analytics before:
<img width="1761" alt="Analytics-before" src="https://user-images.githubusercontent.com/6599111/201685279-63dad20f-e60f-4642-9c60-bda5c5b5ac6d.png">

Analytics after:
<img width="1742" alt="Analytics-after" src="https://user-images.githubusercontent.com/6599111/201685326-ea30681f-3858-4d1e-9db5-ff4a946a4fcd.png">


## Side Effects

Maybe. I didn't have to do as many changes as I thought I was, so this is probably just cosmetic and related to the navigation.

## QA Notes

This shooould be fairly isolated, but it wouldn't hurt to take a look at the affected pages (node analytics, node registrations, and node files) just to make sure everything is fine.


[ENG-4158]: https://openscience.atlassian.net/browse/ENG-4158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
